### PR TITLE
`print_member_value` filter

### PR DESCRIPTION
**Pull request by lquerel**
This PR implements #269

### CODE REVIEW

File: crates/weaver_forge/src/extensions/otel.rs [hunk 1]
```diff
@@ -324,6 +325,19 @@ pub(crate) fn instantiated_type(attr_type: &Value) -> Result<String, minijinja::
     )))
 }
 
+/// Converts an enum member value into:
+/// - A quoted string if the input is a string.
+/// - A non-quoted string if the input is a number or a boolean.
+/// - An empty string otherwise.
+pub(crate) fn print_member_value(input: &Value) -> String {
+    match input.kind() {
+        ValueKind::String => format!("\"{}\"", input),
```
Comment (jsuereth) [inline crates/weaver_forge/src/extensions/otel.rs:334]: Do we need to "escape" quotes in the string when doing this?  Likely...

**Review by jsuereth — APPROVED** [review]
One, somewhat major, nit/fix we'll need to add.

File: crates/weaver_forge/src/extensions/otel.rs (hunk 1 above)
Comment (lquerel) [reply crates/weaver_forge/src/extensions/otel.rs:334]: Oh yes. It was planned and I forgot...

Comment (lquerel) [reply crates/weaver_forge/src/extensions/otel.rs:334]: Fixed

**Pull request merged**